snapshots/overlay/overlay_test.go

```diff
@@ -173,21 +175,14 @@ func testOverlayOverlayMount(t *testing.T, newSnapshotter testsuite.SnapshotterF
 		upper = "upperdir=" + filepath.Join(bp, "fs")
 		lower = "lowerdir=" + getParents(ctx, o, root, "/tmp/layer2")[0]
 	)
-
-	expected := []string{
-		"index=off",
-	}
-	if userxattr, err := NeedsUserXAttr(root); err != nil {
-		t.Fatal(err)
-	} else if userxattr {
-		expected = append(expected, "userxattr")
-	}
-	expected = append(expected, []string{
-		work,
-		upper,
-		lower,
-	}...)
-	for i, v := range expected {
+	var slice []string
```

Comment:
Probably clearer to keep the name as `expected`;

```suggestion
	var expected []string
```